Content.Shared/Anomaly/Components/AnomalyComponent.cs

```diff
@@ -230,16 +230,17 @@ public sealed partial class AnomalyComponent : Component
 /// <summary>
 /// Event raised at regular intervals on an anomaly to do whatever its effect is.
 /// </summary>
+/// <param name="Anomaly">The anomaly pulsing</param>
 /// <param name="Stability"></param>
 /// <param name="Severity"></param>
 [ByRefEvent]
-public readonly record struct AnomalyPulseEvent(float Stability, float Severity);
+public readonly record struct AnomalyPulseEvent(EntityUid Anomaly, float Stability, float Severity);
```

Comment:
the event is raised on the anomaly so you dont need this just use the uid argument from the event handler

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/e282c972-1bc6-4b7e-9ab6-06d16ab3cba3)
![image](https://github.com/space-wizards/space-station-14/assets/96445749/cabb79e2-5de5-4862-8fdf-10e29e54242b)
can't find

Reply:
instead of subscribing to broadcast if you subscribe to it on a component you get entityuid uid parameter to use

Reply:
example pls?